src/runtime/parallel_op.cc

```diff
@@ -62,17 +62,25 @@ Node FFModel::get_or_create_parallel_op_node(
 
   switch (op_type) {
     case OP_COMBINE:
-      return this->get_or_create_combine_node(
-          input, parallel_dim, parallel_degree);
+      CombineParams combine_params;
+      combine_params.combine_legion_dim = parallel_dim;
```

Comment:
```suggestion
      combine_params.legion_dim = parallel_dim;
```
Feels like this would be more concise (not a huge deal either way)

Reply:
All the parallel op files are pretty explicit when prefixing `dim` and `degree` so I followed that pattern for now. However, now that I'm thinking about it, could we have a unified `ParallelOpParams` for combine, replicate, repartition, and reduce? They all have the same structure, and `cached_ops` maintains a separate map for each operator anyway.

Reply:
I think I'd rather keep each separate parallel op having its own params class for now. I'm happy to be convinced that was should go to a single `ParallelOpParams` but either way let's not make it part of this PR.